CMakeLists.txt

```diff
@@ -42,6 +43,10 @@ else()
   set(LINKAGE SHARED)
 endif()
 
+if(CONSTRAINED_DEVICE)
```

Comment:
We might not need this for CPP actually. It should be enough to pass in this value in producer C cmake file right as a cmake build arg? Something like this:

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/CMake/Dependencies/libkvsCommonLws-CMakeLists.txt#L11-L20

So when CPP is run with `cmake .. -DCONSTRAINED_DEVICE=ON`, producer C would add the definition. 
Curious if we tried this.

Reply:
I had expected that as well, but when I tested it it did not work. Perhaps I made a mistake, I'll repeat the test.

Reply:
You're correct, I had misspelled the flag.